topic2_4/target/m2e-wtp/web-resources/META-INF/maven/com.bootcamp.app/topic2_4/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files